projects/packages/forms/src/contact-form/css/grunion.css

```diff
@@ -84,13 +84,21 @@
 	display: block;
 }
 
-.contact-form :where(legend.grunion-field-label),
-.contact-form :where(label) {
+:where(.contact-form legend.grunion-field-label),
```

Comment:
I haven't had a chance to test this yet, I'm sorry, but just thinking that reducing specificity to a root `:where()` can potentially be risky with Classic themes that use a global reset CSS that uses the `legend` selector directly. So the previous `.contact-form :where(legend...)` might work well for Classic themes, but moving all the way to 0 specificity might be a challenge.

Just making a note for myself to test this PR with a couple of Classic themes tomorrow!

Reply:
Oh yeah, classic themes 🤣 

I'll also test this tomorrow. Thank you!

Reply:
hopefully [0791de8](https://github.com/Automattic/jetpack/pull/43545/commits/0791de872986ef27120f82d68a4b545f439ef7ce) fixees that!